src/primitives/block.h

```diff
@@ -25,7 +39,7 @@ class CBlockHeader
 public:
     static const int32_t CURRENT_VERSION=SIZE_FORK_VERSION;
     // This code knows the rules for these block versions:
-    static const int32_t UNDERSTOOD_VERSIONS = (SIZE_FORK_VERSION | 3 | 2 | 1);
+    static const int32_t UNDERSTOOD_VERSIONS = (SIZE_FORK_VERSION | 4 | 3 | 2 | 1);
```

Comment:
The 3 is effectively a comment. It's just indicating that all of the bits that were used for the previous softforks need to be marked as understood versions.